### DISCUSSION TITLE
My vercel account is blocked before even creating

### DISCUSSION THREAD

**Discussion by vishav-singla** [vercel/community#3952] [Account Issue] [answered]
### Question

My vercel account is blocked before even creating. Please unblock it

**Comment by amyegan** [comment]
Sorry you're having trouble signing up for a new account. We can't safely discuss account information in public because of security concerns. You may have received an email with more details about the error and instructions on next steps. 

For more info, please check this guide: [Why am I unable to login or signup to the Vercel platform?](https://vercel.com/guides/why-can-i-not-signup)

**Reply by ScottHamroff** [reply]
I am facing the same issue.

**Comment by vishav-singla** [comment]
Dear Sir,
I haven't got any email from vercel for the next steps. Please unblock my
account with the email: @***@***.*** , I haven't even
ever used it.

**Comment by softkeldozy** [comment]
facing same issue...whats going on here

**Comment by VerdeVistaVoyager** [comment]
Same issue, and no email

**Comment by amyegan** [comment, marked as answer]
Due to the sensitive nature of account information, we cannot safely discuss it here. Please email [registration@vercel.com](mailto:registration@vercel.com) and include the details of the project you want to deploy. That will open a new ticket with the team that can help with account issues. 🙏
